tutorials/7_Advanced_ChangeCode.md

```diff
@@ -67,6 +67,10 @@ The units (e.g., TWa, EJ, GtC, GtCO2, ...) of variables and parameters are docum
 * Don't use $+number combinations, e.g., $20 (this interferes with GAMS error codes).
 * Indicate the end of a file by inserting \*** EOF filename.inc \*** 
 
+#### Sets
```

Comment:
Sure you don't want to put this gem in its own commit?

Reply:
Ah right. That got squashed into another commit when rebasing. I could single it out again.